### DISCUSSION TITLE
캐릭터 움직임이 끊깁니다.

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1116] [5. Q&A]
이전에도 비슷한 포스트가 있었는데 용량을 줄여보라는 권고가 있더군요.
그런데 프로젝트 용량을 줄이라는 의미인건가요?
아니면 씬에 들어가는 게 너무 많아서 생기는 의미인가요?
특별한 건 없고, Navmesh Agent와 particle 정도 쓰고 있습니다.

**Comment by zlmrzlo** [comment]
코드가 정상인데 움직임이 끊겨요? 최적화 과정이 빠진 거 아닌가요?

**Comment by chlguswls32** [comment]
안녕하세요. Sonak0930님.

용량을 줄여보라는 권고가 어떻게 나오는지 궁금하네요. 
GPU, GPU쪽중 어느쪽을 최적화해봐야하는지는 해당 프로젝트를 프로팡일해서 봐야할것같네요.
